src/machines/authMachine.ts

```diff
@@ -31,11 +28,21 @@ export type Events =
     }
 
 export const TOKEN_PERSIST_KEY = 'TOKEN_PERSIST_KEY'
+
+/**
+ * Determine which token do we have persisted to initialize the auth machine
+ */
+const persistedCookie = getCookie(COOKIE_NAME)
+const persistedLocalStorage = localStorage?.getItem(TOKEN_PERSIST_KEY) || ''
+const persistedDevToken = VITE_KC_DEV_TOKEN
 export const persistedToken =
-  VITE_KC_DEV_TOKEN ||
-  getCookie(COOKIE_NAME) ||
-  localStorage?.getItem(TOKEN_PERSIST_KEY) ||
-  ''
+  persistedDevToken || persistedCookie || persistedLocalStorage
+console.log('Initial persisted token')
+console.table([
+  ['cookie', !!persistedCookie],
+  ['local storage', !!persistedLocalStorage],
+  ['dev token', !!persistedDevToken],
```

Comment:
Won't this also be a production token in some cases? Maybe we should keep the name of `persistedToken`. I'm planning to rename `VITE_KC_DEV_TOKEN` to `VITE_KITTYCAD_API_TOKEN` in https://github.com/KittyCAD/modeling-app/pull/7665, FWIW.

Reply:
Ahh yes, this could be a production token if someone takes it from their production account.
I can rename this to api token.

For your fix, could you name it `VITE_KC_API_TOKEN` isntead? to keep the same `KC` semantic?

Reply:
The rust libraries seem to pick up `KITTYCAD_API_TOKEN` automatically so I'd like to match that convention to reduce the number of variables we have to set. Feel free to chime in on that PR when it's out of draft! 😄